sklearn/linear_model/tests/test_base.py

```diff
@@ -677,7 +686,8 @@ def test_dtype_preprocess_data():
 
 
 @pytest.mark.parametrize("n_targets", [None, 2])
-def test_rescale_data_dense(n_targets):
+def test_rescale_data_dense(n_targets, global_random_seed):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```